tekton/cronjobs/resource-cd-cron-base/trigger-resource-cd.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: resource-cd-trigger
+spec:
+  schedule: "12 * * * *" # Houly at *:12
```

Comment:
seems like we're a bit inconsistent about whether we put a comment to explain the cron expression?

*nittiest nit pick*

Reply:
heh, upz